app/views/help.scala.html

```diff
@@ -100,146 +250,203 @@ <h3 class="govuk-heading-s">Step two: Transfer agreement</h3>
+                        Knowing the format also aids TNA with preservation decisions about the record.
+                        We accept most digital formats, but ones we haven’t seen before may need further investigation.
+                        We may contact you if your transfer contains file formats that we can’t identify.
                     </p>
-                <li class="govuk-!-font-weight-bold">Validating data integrity</li>
+                    <li>Validating data integrity</li>
                     <p class="govuk-body">
-                        A check performed to ensure files have not been corrupted as they move between your shared drive and TDR during the upload process. The data integrity check uses checksums which are like a digital fingerprint used to ensure the data has not changed. For TDR we use SHA256.
+                        A check performed to ensure files have not been corrupted as they move between your shared drive and TDR during the upload process.
+                        The data integrity check uses checksums which are like a digital fingerprint used to ensure the data has not changed.
+                        For TDR we use SHA256.
                     </p>
-            </ul>
-            <p class="govuk-body">
-                If your files fail TDR checks, we will inform you. Failing checks may mean you cannot proceed with your transfer until the problems are resolved.
-            </p>
-            <h3 class="govuk-heading-s">Step five: Add descriptive metadata</h3>
-            <p class="govuk-body">
-                Although TDR runs DROID to capture basic file-level metadata during the upload, you should provide additional information about your records if you can. This can include anything from descriptions to additional dates. Any additional information that will provide context and richness to the record and make it easier to find and understand in the future.
+                </ul>
+                <p class="govuk-body">The checking process is likely to take several minutes.</p>
+                <img src="@routes.Assets.versioned("images/standard-checks-complete.png")" class="screenshots" alt="Checks complete page" />
+                <li>On completion, you will be advised on screen when you can proceed to the next step.
+                    Click 'Continue' to see the results of your checks.</li>
+                <img src="@routes.Assets.versioned("images/standard-results-check.png")" class="screenshots" alt="Results of checks page" />
+                <li>You will be informed on screen of the results of your checks. </li>
+                <li>If your checks have been successful, click 'Continue' to progress to the next step.</li>
+            </ol>
+            <p class="govuk-body">If your files fail TDR checks, we will inform you.
+                Failing checks may mean you cannot proceed with your transfer until the problems are resolved.
```

Comment:
Should it be "Failing checks *will* mean" instead of "Failing checks *may* mean"? Or are there actually some failed checks that we will allow?

Reply:
At some future point the users will be able to remove files that fail from the consignment for certain reasons. I think this is fine to leave as is for now.